tests/performance-test/deploy/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+/usr/sbin/collectd -C /tmp/minimal-collectd.conf -f 2>&1 | tee /tmp/collectd_output
+
+sleep 5
```

Comment:
I came back to this PR to answer the questions in my mind "How does the events pod get shut down?" because I missed it the first time through. I see that this sleep co-ordinates with a `sleep 3` below while you wait for collectd to shut down then copy the file out before the pod terminates. Tricky! This is probably safe enough for our purposes but try to keep co-ordinate-by-sleep out of anything that runs in production.

Reply:
For sure. The problem here is that this script is limited to using the 'oc' tool which doesn't give all of the controller ability that a native python or golang api provides, so I had to resort to some hacks